src/ui/screens/SubIdentities/AddSubIdentity.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {render, fireEvent} from 'src/testUtils';
+import {AddSubIdentity} from './AddSubIdentity';
+
+jest.useFakeTimers();
```

Comment:
Why do you need this?

Reply:
Initially, when I was writing a test case using the `test` function, I had an import teardown, so adding this resolved it. Now I'm testing within describe, so I don't need it. It was removed in the latest push.